docs-src/ref-error-codes.scrbl

```diff
@@ -0,0 +1,334 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[#:version reach-vers #:tag "ref-error-codes"]{Error Codes}
+
+This section provides an in depth explanation of the error codes produced from
+the Reach compiler.
+
+@;{
+  What is a "good" error description?
+  * Intuitive explanation
+  * Example errorneous program
+  * How to fix to fix that program
+  * Any other advice
+}
+
+@section{RE0001}
```

Comment:
You should make `@error` or something like `(define (error x) @section[#:tag x]{@|x|})`